tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -110,9 +110,15 @@ stages:
     DoCompliance: ${{ parameters.DoCompliance }}
     DoEsrp: ${{ parameters.DoEsrp }}
     IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
-    OrtNugetPackageId: 'Microsoft.ML.OnnxRuntime${{ parameters.NugetPackageSuffix }}'
+    ${{ if not(eq(parameters.NugetPackageSuffix, 'NONE')) }}:
```

Comment:
You may use "ne" instead of "not(eq(...))" or swap the two statements to make the conditional statement easier to understand.

Reply:
Done - used "eq" instead.